exporters/metrics/src/main/java/com.google.cloud.opentelemetry.metric/MetricTranslator.java

```diff
@@ -45,20 +49,28 @@
   static final Set<MetricData.Descriptor.Type> DOUBLE_TYPES = ImmutableSet
       .of(NON_MONOTONIC_DOUBLE, MONOTONIC_DOUBLE);
 
+  static final String GCP_HOST_ID_KEY = "instance_id";
+  static final String GCP_ACCOUNT_ID_KEY = "project_id";
+  static final String GCP_ZONE_KEY = "zone";
+  static final String GCP_CLUSTER_NAME_KEY = "cluster_name";
+  static final String GCP_NAMESPACE_KEY = "namespace_id";
+  static final String GCP_POD_KEY = "pod_id";
+  static final String GCP_CONTAINER_KEY = "container_name";
+
   static final Map<String, Map<String, String>> OTEL_TO_GCP_LABELS = ImmutableMap.<String, Map<String, String>>builder()
       .put("gce_instance", ImmutableMap.<String, String>builder()
-          .put("host.id", "instance_id")
-          .put("cloud.account.id", "project_id")
-          .put("cloud.zone", "zone")
+          .put("host.id", GCP_HOST_ID_KEY)
```

Comment:
Could you also define `OTEL_HOST_ID_KEY` and similar for keys?
They should be defined in the SDK, but for now having a single place with constants would be still useful.

Reply:
As mentioned in the other comment, might be better to just remove this mapping code for now, and add it separately later.